analyzers/Threatcrowd/threatcrowd_analyzer.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+import requests
+from cortexutils.analyzer import Analyzer
+
+class Threatcrowd(Analyzer):
+
+    URI = "https://www.threatcrowd.org/searchApi/v2"
+
+    def summary(self, raw):
+        taxonomies = []
+
+        level = "info"
+        value = "None"
+
+        if 'votes' in raw:
+            r = raw.get('votes')
+            value = r
```

Comment:
`r` is `None` if ThreatCrowd doesn't know an email. That should be catched somehow.

Reply:
(the `level` would be unspecified that way)